invenio_vocabularies/contrib/subjects/jsonschemas/subjects/subject-v1.0.0.json

```diff
@@ -1,22 +1,30 @@
 {
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "local://subjects/subject-v1.0.0.json",
   "description": "Subjects from specific vocabularies.",
+  "type": "object",
+  "additionalProperties": false,
   "properties": {
-    "identifier": {
-      "description": "Subjects term identifier.",
-      "type": "string"
+    "$schema": {
+      "$ref": "local://definitions-v1.0.0.json#/$schema"
     },
-    "scheme": {
-      "description": "Identifier scheme.",
-      "type": "string"
+    "id": {
+      "description": "Subject unique identifier.",
+      "$ref": "local://definitions-v1.0.0.json#/identifier"
     },
-    "term": {
-      "description": "Subject term value.",
+    "name": {
+      "description": "Alternative to id. Custom free-text keyword.",
```

Comment:
Like was brought up, I am not sure anymore if we even want to store custom keywords here actually. Just storing them on the record is fine.